cli/data.go

```diff
@@ -202,7 +208,7 @@ func downloadBinary(ctx context.Context, client datapb.DataServiceClient, dst, i
 	}
 
 	//nolint:gosec
-	dataFile, err := os.Create(filepath.Join(dst, "data", datum.GetMetadata().GetId()+datum.GetMetadata().GetFileExt()))
+	dataFile, err := os.Create(filepath.Join(dst, dataDir, "data"+".ndjson"))
```

Comment:
Think this got copied from tabular, but this would instead be `dataFile, err := os.Create(filepath.Join(dst, dataDir, fileName+datum.GetMetadata().GetFileExt()))` in order to reuse the timestamped binary file and its file extension

Reply:
Eek yes bad copying/pasting